src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
this is still a BC break for any project registering a custom validator for this alias. Changing FrameworkBundle is not removing the BC break.